cmd/resim/commands/batch.go

```diff
@@ -346,6 +351,93 @@ func createBatch(ccmd *cobra.Command, args []string) {
 	}
 }
 
+func batchToSlackWebhookPayload(batch *api.Batch) *slack.WebhookMessage {
+	var (
+		suite   *api.TestSuite
+		system  *api.System
+		reports *[]api.Report
+	)
+	// todo: matt: programatically convert api url from cobra to frontend
+	baseUrl := fmt.Sprintf("%sprojects/%s", "https://app.resim.io/", batch.ProjectID.String())
+	blocks := &slack.Blocks{BlockSet: make([]slack.Block, 3, 3)}
+
+	// Get the suite object
+	suiteResponse, err := Client.GetTestSuiteWithResponse(context.Background(), *batch.ProjectID, *batch.TestSuiteID)
+	if err != nil {
+		log.Fatal("unable to retrieve suite for batch:", err)
+	}
+	ValidateResponse(http.StatusOK, "unable to retrieve suite for batch", suiteResponse.HTTPResponse, suiteResponse.Body)
+	suite = suiteResponse.JSON200
```

Comment:
```suggestion
	if suiteResponse.JSON200 == nil {
	   log.Fatal(...)
	}
	testSuite := *suiteResponse.JSON200
```
and already dereference so there are no pointers hanging around

Reply:
ditto others